### ISSUE TITLE
Suggested apps are always the same

### ISSUE THREAD

**Issue by Mo7y** [amirzaidi/Launcher3#41]
First, thanks for all the efforts! Using your Launcher3 v1.3, on Nexus 6P, Android O DP3 8.0.0, the Suggested/favorite apps row on top of the app drawer is always showing the same applications, regardless what apps I recently used or what I use more often than other apps.
Edit: To clarify the issue, what I recall from the official Pixel Launcher when I tried it, is that the favorite apps are regularly updated on top of the app drawer to show the most used apps recently.

**Comment by RWOne** [comment]
Hello,

First off, congrats on awesome work!

Nexus 6, stock,only unlocked bootloader, running 7.1.1 (N6F27E).

Love it so far!

Adding that I see the same thing on 1.3.  However let's make sure terminology is correct.
The top row od Suggested apps remains static (I have toggled the option off then on, then ran a few hours last night opening several apps including Google ones, such as maps camera, g+, etc.)

The top row remains at Youtube, Chrome, Play music, Calendar (Though date changes, thanks!), and News and weather.

Not to hijack, but something else I noticed. The google app/now pane, on your launcher, I suddenly got the search shortcuts this morning, never had them before on the google now launcher, I switched back to that for an hour, no change, no shortcuts, back to yours and they're there. I think Google is playing with me, as I don't believe you have anything special running in this regard, but wanted to note it.

If anything it's just another reason to use it!  :)

Thanks again for the hard work, looking forward to future releases, IMHO, I'd like to see all the app  settings options that the actual PL has, it seems you have hit all the features already.

Just awesome ....  :)

**Comment by amirzaidi** [comment]
I'm aware of this issue and will look into fixing it. There's a lot of unstable code pulled in to even get it to work in the first place, trust me I really want to fix this too. It's the reason I didn't initially include it before 1.3.

**Comment by RWOne** [comment]
Amirzqidi,

Figured you were on it. Just wanted to give a more complete report.

Suggested apps not a deal breaker for most :)

Thanks for the gift (of the launcher)!

**Comment by RWOne** [comment]
Sorry for the mangled name, Gboard ... Ugh.

**Comment by RWOne** [comment]
Sorry, mind just working today ...

Could it be something to do with that there are two locations for toggling suggested apps?
One in your settings for the launcher, and I found by going to Settings -> Google -> Search, then look under home screen subtitle (At least on my N6 phone on 7.1.1) there is another, and it seems independent from the one for the launcher.

Just both asking, and thought you would like to know what I found there.  :)

Enjoy your week!
